wireshark/USAGE.md

```diff
@@ -0,0 +1,39 @@
+# OQS-wireshark
+
+This docker image contains a version of [wireshark](https://www.wireshark.org/) built to also properly display quantum-safe crypto (QSC) TLS operations.
+
+To this end, it contains references to algorithms supported by [liboqs](https://github.com/open-quantum-safe/liboqs) and [OQS-OpenSSL](https://github.com/open-quantum-safe/openssl) from the [OpenQuantumSafe](https://openquantumsafe.org) project.
+
+The image is based on Ubuntu and requires the host to run the Unix X-Window system.
+
+## Quick start
+
+Execute this command to open the wireshark window on your host:
+
+    docker run --net=host --privileged --env="DISPLAY" --volume="$HOME/.Xauthority:/root/.Xauthority:rw" openquantumsafe/wireshark
+
+*Note:* Select a network interface to monitor.
+
```

Comment:
I had to do the following on Ubuntu to make it run:
```suggestion

You may need to grant permissions for Docker to access the X display:

    xhost +si:localuser:$USER

```

Reply:
That should not be necessary: The .Xauthority volume mount should do the same trick. But then again, different distributions may do things differently... So added. Thx.